lib/battle_boats/board.rb

```diff
@@ -66,6 +66,10 @@ def cell_at(coordinate:)
       end
     end
 
+    def get_random_coordinate
+      BattleBoats::Coordinate.random(row: 0..9, column: 0..9)
```

Comment:
What concept does `0` and `9` represent? Is there away to name these numbers/range of numbers?

Reply:
Good idea! The "magic number" `10` is encoded in a few places: `Array.new(10)` in `create_play_area`, and ranges in `within_range` and `get_random_coordinate`. There's definitely a name for what this number represents :smiles: